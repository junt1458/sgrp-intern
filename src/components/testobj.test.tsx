import TestObject from './testobj';
import { render } from '@testing-library/react';

it('Test render', () => {
  const { container } = render(<TestObject />);
  expect(container).toMatchSnapshot();
});
