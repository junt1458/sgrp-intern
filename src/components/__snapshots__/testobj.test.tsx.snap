// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render 1`] = `
<div>
  test
</div>
`;
